### ISSUE TITLE
[bug] Row changes are not reflected in the table

### ISSUE THREAD

**Issue by happyDemon** [xaksis/vue-good-table#23]
When I update properties on a row, these changes are not reflected in the table.
It ends up with stale data.

**Comment by happyDemon** [comment]
This is because line 264

```javascript
        var computedRows = JSON.parse(JSON.stringify(this.rows));
```

**Comment by xaksis** [comment]
Hmm, interesting... I'll try to recreate this. I have a watcher on rows setup to watch for changes as well. Thank you for pointing it out.

**Comment by happyDemon** [comment]
I've been digging in a little bit deeper, it might be better to make the  filteredRows a computed property?

**Comment by xaksis** [comment]
ah okay... so the problem was that my watcher isn't set up to deep watch and since rows is an array of objects we need to deep watch for this to work. Thank you again for pointing this out. 

making filteredRows a computed property won't help I think because processedRows already have filteredRows as a dependency and filteredRows are updated everytime row is updated. so the chain is: rows -> filteredRows -> processedRows -> paginated

I'll check in the fix shortly.

**Comment by xaksis** [comment]
fixed: v1.6.3
